<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File scanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="btnSelectFolder">
        <property name="text">
         <string>Select folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnScan">
        <property name="text">
         <string>Scan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExport">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblProgress">
      <property name="text">
       <string>Found: 0 files, Size: 0 MB</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableFiles">
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupStats">
      <property name="title">
       <string>Basic statistic</string>
      </property>
      <layout class="QVBoxLayout" name="verticalStats">
       <item>
        <widget class="QLabel" name="lblTotalFiles">
         <property name="text">
          <string>Total files: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblTotalSize">
         <property name="text">
          <string>Total size: 0 MB</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblLargestFile">
         <property name="text">
          <string>Largest file: -</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblMostCommon">
         <property name="text">
          <string>The most common type: -</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblStatus">
      <property name="text">
       <string>Status: Waiting</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
